app/utils/query_helpers.py

```diff
@@ -116,31 +117,39 @@ def query_count(query: Query) -> int:
 
 
 @functools.cache
-def get_stat_domains() -> dict[str, type[DeclarativeBase]]:
-    return {
-        "overall": DeadlockedOverallStats,
-        "deathmatch": DeadlockedDeathmatchStats,
-        "conquest": DeadlockedConquestStats,
-        "ctf": DeadlockedCTFStats,
-        "koth": DeadlockedKOTHStats,
-        "juggernaut": DeadlockedJuggernautStats,
-        "weapon": DeadlockedWeaponStats,
-        "vehicle": DeadlockedVehicleStats,
-
-        "horizon": DeadlockedHorizonStats,
-        "snd": DeadlockedSNDStats,
-        "payload": DeadlockedPayloadStats,
-        "spleef": DeadlockedSpleefStats,
-        "infected": DeadlockedInfectedStats,
-        "gungame": DeadlockedGungameStats,
-        "infinite_climber": DeadlockedInfiniteClimberStats,
-        "survival": DeadlockedSurvivalStats,
-        "survival_orxon": DeadlockedSurvivalOrxonStats,
-        "survival_mountain_pass": DeadlockedSurvivalMountainPassStats,
-        "survival_veldin": DeadlockedSurvivalVeldinStats,
-        "training": DeadlockedTrainingStats,
-    }
-
+def get_stat_domains(game: str) -> dict[str, type[DeclarativeBase]]:
+    if game == 'dl':
+        return {
+            "overall": DeadlockedOverallStats,
+            "deathmatch": DeadlockedDeathmatchStats,
+            "conquest": DeadlockedConquestStats,
+            "ctf": DeadlockedCTFStats,
+            "koth": DeadlockedKOTHStats,
+            "juggernaut": DeadlockedJuggernautStats,
+            "weapon": DeadlockedWeaponStats,
+            "vehicle": DeadlockedVehicleStats,
+
+            "horizon": DeadlockedHorizonStats,
+            "snd": DeadlockedSNDStats,
+            "payload": DeadlockedPayloadStats,
+            "spleef": DeadlockedSpleefStats,
+            "infected": DeadlockedInfectedStats,
+            "gungame": DeadlockedGungameStats,
+            "infinite_climber": DeadlockedInfiniteClimberStats,
+            "survival": DeadlockedSurvivalStats,
+            "survival_orxon": DeadlockedSurvivalOrxonStats,
+            "survival_mountain_pass": DeadlockedSurvivalMountainPassStats,
+            "survival_veldin": DeadlockedSurvivalVeldinStats,
+            "training": DeadlockedTrainingStats,
+        }
+    elif game == 'uya':
+        return {
+            "overall": UyaOverallStats,
+            "deathmatch": UyaDeathmatchStats,
+            "siege": UyaSiegeStats,
+            "ctf": UyaCTFStats,
+            "weapon": UyaWeaponStats,
+        }
```

Comment:
I would convert this to a dictionary with the game input as the key. That way if an incorrect key is passed it will raise a KeyError in the chain and point out this function instead of returning None and requiring the developer to backtrack to this function.